Containers/mastercontainer/cron.sh

```diff
@@ -47,8 +47,14 @@ while true; do
     # Remove sessions older than 24h
     find "/mnt/docker-aio-config/session/" -mindepth 1 -mmin +1440 -delete
 
+    # Remove nextcloud-aio-domaincheck container
+    sudo -u www-data docker container remove nextcloud-aio-domaincheck --force
+
     # Remove dangling images
-    sudo -u www-data docker image prune -f
+    sudo -u www-data docker image prune --force
+    
+    # Remove dangling volumes
+    for volume in $(sudo -u www-data docker volume ls --quiet --filter dangling=true | grep -E "^[a-z0-9]{64}$"); do sudo -u www-data docker volume remove "$volume"; done
```

Comment:
I am honestly against removing any volumes automatically.

The better way is to remove anonymous volumes when the container gets removed which we already do in https://github.com/nextcloud/all-in-one/blob/3f22ec8b4d26cb3b8275cd7ae8a71ba2c957f067/php/src/Docker/DockerActionManager.php#L148